Makefile

```diff
@@ -43,7 +43,9 @@ wait-and-validate:
 	@echo "Validating"
 	SKIP_TESTS="$(SKIP_TESTS)" DONT_FOCUS=true TEST_SUITES="validationsuite" hack/run-functests.sh
 
-functests-on-ci: setup-test-cluster feature-deploy feature-wait functests
+functests-on-ci: setup-test-cluster setup-build-index-image feature-deploy feature-wait functests
+
+functests-on-internal-ci: setup-test-cluster feature-deploy feature-wait functests
```

Comment:
mind changing the name to something that reflects the behaviour and not where this is supposed to run?
like functests-on-ci-no-index or something like that